src/dotnet-serve/SimpleServer.cs

```diff
@@ -52,12 +51,7 @@ public SimpleServer(CommandLineOptions options, IConsole console)
 
             var path = directory != null
                 ? Path.GetFullPath(directory)
-                : Directory.GetCurrentDirectory();
-
-            if (!string.IsNullOrEmpty(pathBase) && pathBase[0] != '/')
-            {
-                pathBase = "/" + pathBase;
-            }
```

Comment:
I moved the preceding slash test for path base out of `SimpleServer` and into the `CommandLineOptions` class as a method. It looks like this got lost in the big refactoring you did (`pathBase` wasn't actually used anywhere). Not strictly related to this PR, but I noticed it while I was in there rebasing/refactoring.

Reply:
Thanks. Guess i should write tests for this app or something